index.js

```diff
@@ -113,6 +113,7 @@ MPlayer.prototype = _.extend({
     },
     stop: function() {
         this.player.cmd('stop');
+        this.status.playing = false;
```

Comment:
Let's change this to `this.setStatus({playing: true});` for convention's sake :)